### PR TITLE
Add sanity integration test to check minimal configuration

### PR DESCRIPTION
**Pull request by rpanchyk**
We need an application test which shows PBS starts up with default config params without errors.
This will get us on track with documentation and helps project to be more friendly.

### CODE REVIEW

File: src/test/java/org/prebid/server/it/SanityTest.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import org.junit.runner.RunWith;
+import org.prebid.server.VertxTest;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)
+@TestPropertySource(properties = {
+        "spring.config.additional-location=sample-prebid-config.yaml",
+        "http.port=8081",
```
Comment (schernysh) [inline src/test/java/org/prebid/server/it/SanityTest.java]: Why not include these values in the config file?

File: sample-prebid-config.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+adapters:
```
Comment (schernysh) [inline sample-prebid-config.yaml]: What about moving these two sample files to `sample` directory or alike?

Comment (rpanchyk) [reply sample-prebid-config.yaml]: The idea was to make it simpler for newcomer for quick start, so it is better these files be in root folder.

File: src/test/java/org/prebid/server/it/SanityTest.java (hunk 1 above)
Comment (rpanchyk) [reply src/test/java/org/prebid/server/it/SanityTest.java]: Actually, this is dirty hack for JUnit - sanity test starts before previous integration test finished. So, i obtained binding port error. For now i don't have a solution and just override default ports.

**Pull request merged**